networkx/algorithms/shortest_paths/weighted.py

```diff
@@ -220,7 +223,7 @@ def dijkstra_path_length(G, source, target, weight='weight'):
             "node %s not reachable from %s" % (source, target))
 
 
-def single_source_dijkstra_path(G, source, cutoff=None, weight='weight'):
+def single_source_dijkstra_path(G, source, *args, **kw):
```

Comment:
I'd prefer not to use *args and **kw.  It makes the functions harder to understand and can be a cause for hidden errors due to unprocessed arguments.